nsls2tools/csx1/startup/settings.py

```diff
@@ -0,0 +1,95 @@
+from bluesky.magics import BlueskyMagics
+
+from .startup import sd
+from .detectors import *
+from .endstation import *
+from .accelerator import *
+from .optics import *
+from .tardis import *
+
+#
+# Setup of sup. data for plans
+#
+sd.monitors = []
+sd.flyers = []
+sd.baseline = [theta, delta, gamma,
+               sx, say, saz,
+               cryoangle, sy, sz,
+               epu1, epu2,
+               slt1, slt2, slt3,
+               m1a, m3a,
+               nanop, tardis,
+               stemp, pgm]
+
+sclr.names.read_attrs=['name1','name2','name3','name4','name5','name6']  # TODO  WHAT IS THIS??? - Dan Allan
```

Comment:
See my comment in the code. ;- )

Reply:
@danielballan This was not my question.  But I do not see your answer in the code directly above or below.

Reply:
you mean 
```
+# This was imported in 00-startup.py #  used to generate the list: [thing.name for thing in get_all_positioners()]```